python/lsst/sims/photUtils/Sed.py

```diff
@@ -106,6 +106,10 @@ def __init__(self, wavelen=None, flambda=None, fnu=None, badval=numpy.NaN, name=
         self.zp = -2.5*numpy.log10(3631)
         self.name = name
         self.badval = badval
+
+        self._physParams = PhysicalParameters()
+        self._seedingDefaults = LSSTdefaults()
```

Comment:
Is this a typo? (seedingDefaults is not used anywhere else, but _seeingDefaults is. 
Does LSSTdefaults only carry the seeing default values (at least, those are the only parameters relevant for this class?)